spec/dummy/db/seeds.rb

```diff
@@ -0,0 +1,5 @@
+Caffeine::AdminUser.create(email: 'admin@example.com', password: 'q1w2e3')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.